app/main.py

```diff
@@ -71,12 +71,45 @@ def move():
     # debug board object boards
     board_object.print_int_board()
     board_object.print_dima_board()
+    try:
+        direction = cardinal(converted_data, target_selection(converted_data,
+                                                              board_object))
+        response = {"move": direction, "shout": board_object.food_string()}
+    except Exception:
```

Comment:
You're specifically only catching 'Exceptions'.  To my knowledge, this won't catch other things like NameErrors, and LookupErrors.

I think you should either have a blanket except under the filtered one. Or remove 'Exception' to have it catch all errors.